src/openrct2/world/Banner.h

```diff
@@ -43,10 +43,8 @@ struct Banner
     }
 
     std::string GetText() const;
-    void FormatTextTo(Formatter&, bool addColour) const;
-    void FormatTextTo(Formatter&) const;
-    size_t FormatTextTo(void* args, bool addColour) const;
-    size_t FormatTextTo(void* args) const;
+    size_t FormatTextTo(Formatter&, bool addColour) const;
+    size_t FormatTextTo(Formatter&) const;
```

Comment:
```suggestion
    void FormatTextTo(Formatter&, bool addColour) const;
    void FormatTextTo(Formatter&) const;
```

No one is using the return values so either this can be `void` or something is amiss up there.